tools/print_all_namespace.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Traceability. What does this have to do with the component diagram? Please create a separate issue and adjust the commit message iox number to be traceable.

Reply:
Nope, script and diagram belong together. This script can be used to update the diagram and make sure not namespaces are missing. We could also automate this at some point.